docs/website/docs/plus/getting-started/tutorial.md

```diff
@@ -0,0 +1,385 @@
+
+* initialize a new dlt+ Project
+* navigate the `dlt.yml` file
+* add sources, destinations, and pipelines
+* run pipelines using cli commands
+* inspect datasets
+* work with dlt+ Profiles for enabling different configurations
+
+## Prerequisites
+
+- Python 3.9 or higher installed
+- Virtual environment with installed dlt+ package ([installation guide](./installation.md))
+- Valid dlt+ license (EULA)
+- Make sure you're familiar with the [core concepts of dlt](../../reference/explainers/how-dlt-works.md)
+
+:::tip
+You can find the full list of available cli commands under [cli reference](../reference.md)
+:::
+
+## Creating a new dlt+ Project
+
+### Generating your first project
+Start by creating a new folder for your project. Then, navigate to the folder in your terminal.
+
+```sh
+mkdir tutorial && cd tutorial
+```
+
+Run the following command to initialize a new dlt+ Project:
+
+```sh
+# Initialize a dlt+ Project named "tutorial", the name is derived from the folder name
+dlt project init arrow duckdb
+```
+
+This command generates a project named `tutorial` with:
+- one [pipeline](../../general-usage/pipeline)
+- one Arrow source defined in `sources/arrow.py`
+- one DuckDB destination
+- one dataset on the DuckDB destination
+
+### The generated folder structure
+After running the command, the following folder structure is created:
+
+```sh
+.
+├── .dlt/                 # your dlt settings including profile settings
+│   ├── config.toml
+│   ├── dev.secrets.toml
+│   └── secrets.toml
+├── _storage/             # local storage for your project, excluded from git
+├── destinations/         # your destinations, empty in this example
+├── sources/              # your sources, contains the code for the arrow source
+│   └── arrow.py
+├── .gitignore
+└── dlt.yml       # the main project manifest
+```
+
+### Understanding `dlt.yml`
+The `dlt.yml` file is the central configuration for your dlt+ Project. It defines the pipelines, sources, and destinations. In the generated project, the file looks like this:
+
+```yaml
+# project settings
+project:
+  name: template
+
+profiles:
+  # profiles allow you to configure different settings for different environments
+  dev: {}
+
+# your sources are the data sources you want to load from
+sources:
+  arrow:
+    type: sources.arrow.source
+
+# your destinations are the databases where your data will be saved
+destinations:
+  duckdb:
+    type: duckdb
+
+# your datasets are the datasets on your destinations where your data will go
+datasets: {}
+
+# your pipelines orchestrate data loading actions
+pipelines:
+  my_pipeline:
+    source: arrow
+    destination: duckdb
+    dataset_name: my_pipeline_dataset
+```
+
+:::tip
+If you do not want to start with a source, destination, and pipeline, you can simply run `dlt project init tutorial`, where `tutorial` is the name of your dlt+ Project. This will generate a project with empty sources, destinations, and pipelines.
+:::
```

Comment:
we should write some sort of warning that `dlt project init` doesn't work properly (or at all?) with verified sources right now. But that it will be implemented soon

Reply:
Added warning, when we first introducing the init